ddtrace/debugging/_probe/remoteconfig.py

```diff
@@ -236,7 +236,7 @@ def _update_probes_for_config(self, config_id, config):
         # type: (str, Any) -> None
         prev_probes = self._configs.get(config_id, {})  # type: Dict[str, Probe]
         next_probes = (
-            {probe.probe_id: probe for probe in get_probes(config_id, config)} if config is not None else {}
+            {probe.probe_id: probe for probe in get_probes(config_id, config)} if config not in (None, False) else {}
```

Comment:
What the semantic difference between False and None? 
It look like a broken API for _update_probes_for_config callback.

Reply:
The typing of the expected signature of the callback mechanism in the RCM client is a bit funny at the moment. I believe `False` is being (ab)used as a sentinel object to indicate disabling of configuration items 😞 .